src/main/resources/avro/rnaquantificationmethods.avdl

```diff
@@ -0,0 +1,210 @@
+@namespace("org.ga4gh.methods")
+protocol RnaQuantificationMethods {
+
+import idl "methods.avdl";
+import idl "common.avdl";
+import idl "rnaquantification.avdl";
+
+/****************  /rnaquantification/search  *******************/
+/**
+This request maps to the body of 'POST /rnaquantification/search'
+as JSON.
+*/
+record SearchRnaQuantificationRequest {
+  /**
+  If present, return only Rna Quantifications which belong to this set.
+  */
+  union { null, string } rnaQuantificationId = null;
+
+  /**
+  If present, return only Rna Quantifications which belong to this set.
+  */
+  union { null, string } datasetId = null;
+
+  /**
+  Specifies the maximum number of results to return in a single page.
+  If unspecified, a system default will be used.
+  */
+  union { null, int } pageSize = null;
+
+  /**
+  The continuation token, which is used to page through large result sets.
+  To get the next page of results, set this parameter to the value of
+  'nextPageToken' from the previous response.
+  */
+  union { null, string } pageToken = null;
+}
+
+/**
+This is the response from 'POST /rnaquantification/search' expressed as JSON.
+*/
+record SearchRnaQuantificationResponse {
+  /** The list of matching quantifications. */
+  array<org.ga4gh.models.RnaQuantification> rnaQuantification = [];
+
+  /**
+  The continuation token, which is used to page through large result sets.
+  To get the next page of results, set this parameter to the value of
+  'nextPageToken' from the previous response.
+  */
+  union { null, string } nextPageToken = null;
+}
+
+/**
+Gets a list of 'RnaQuantification' matching the search criteria.
+
+'POST /rnaquantification/search' must accept JSON version of
+'SearchRnaQuantificationRequest' as the post body and will return a JSON
+version of 'SearchRnaQuantificationResponse'.
+*/
+SearchRnaQuantificationResponse searchRnaQuantification(
+    /**
+    This request maps to the body of 'POST /rnaquantification/search'
+    as JSON.
+    */
+    SearchRnaQuantificationRequest request) throws GAException;
+
+/**************** /rnaquantification/{id} *******************/
+/**
+Gets a `RnaQuantification` by ID.
+`GET /rnaquantification/{id}` will return a JSON version of `RnaQuantification`.
+*/
+org.ga4gh.models.RnaQuantification getRnaQuantification(
+  /**
+  The ID of the `RnaQuantification`.
+  */
+  string id) throws GAException;
+
+/****************  /expressionlevel/search  *******************/
+/**
+This request maps to the body of 'POST /expressionlevel/search'
+as JSON.
+*/
+record SearchExpressionLevelRequest {
+  /**
+  If present, return matching Expression Level record.
+  */
+  union { null, string } expressionLevelId = null;
+
+  /**
+  If present return only ExpressionLevel records which belong to this set.
+  */
+  union { null, string } featureGroupId = null;
+
+  /**
+  The rnaQuantification to restrict search to.
+  */
+  string rnaQuantificationId = null;
```

Comment:
This either needs to be an empty string default, or the type needs to be a union with null.